src/lib/bcmp/bcmp_config.cpp

```diff
@@ -145,44 +143,23 @@ bool bcmp_config_status_response(uint64_t target_node_id,
       rval = true;
     }
   } while (0);
-  vPortFree(status_resp_msg);
+  bm_free(status_resp_msg);
   return rval;
 }
 
-static void bcmp_config_process_commit_msg(bm_common_config_commit_t *msg) {
+static void bcmp_config_process_commit_msg(BmConfigCommit *msg) {
   configASSERT(msg);
-  switch (msg->partition) {
-  case BM_COMMON_CFG_PARTITION_USER: {
-    USR_CFG->saveConfig(); // Reboot!
-    break;
-  }
-  case BM_COMMON_CFG_PARTITION_SYSTEM: {
-    SYS_CFG->saveConfig(); // Reboot!
-    break;
-  }
-  default:
-    printf("Invalid partition\n");
-    break;
-  }
+  bcmp_commit_config((BmConfigPartition)msg->partition);
```

Comment:
the switch statement is moved into bcmp_commit_config, which is defined in `bm_config_wrapper.cpp`. This is the same for the other processing functions, I moved the part that does the "config stuff" into the wrapper file.